src/site/content/en/blog/when-to-use-local-https/index.md

```diff
@@ -29,7 +31,7 @@ However, in the following cases, you'll need HTTPS for local development:
     <figcaption class="w-figcaption">When to use HTTPS for local development.</figcaption>
   </figure>
 
-See [How to use HTTPS for local development](/how-to-use-local-https).
+If you're in this case, check **[How to use HTTPS for local development](/how-to-use-local-https)**.
```

Comment:
```suggestion
{% Aside %}
If you need to use HTTPS for one of the above use cases, check out **[How to use HTTPS for local development](/how-to-use-local-https)**.
{% endAside %}
```

Reply:
Links already have plenty of emphasis (color, underline) so bolding them doesn't really do much. I think this will make it stand out more.

Reply:
Great idea thanks